src/test/java/part3/exercise/lambda/LambdaExercise.java

```diff
@@ -38,19 +43,20 @@ public void function() {
 
     @Test
     public void combineFunctions() {
-        final Function<Person, String> getPersonName = null; // TODO get the name of person
+        final Function<Person, String> getPersonName = Person::getFirstName; // TODO get the name of person
 
         assertEquals("John", getPersonName.apply(new Person("John", "Galt", 30)));
 
-        final Function<String, Integer> getStringLength = null; // TODO get string length
+        final Function<String, Integer> getStringLength = (a) -> a.length(); // TODO get string length
 
         assertEquals(Integer.valueOf(3), getStringLength.apply("ABC"));
 
         // TODO get person name length using getPersonName and getStringLength without andThen
-        final Function<Person, Integer> getPersonNameLength1 = null;
+        final Function<Person, Integer> getPersonNameLength1 = person1 -> person1.getFirstName().length();

```

Comment:
Note `// TODO get person name length using getPersonName and getStringLength without andThen` above.